subxt/src/backend/unstable/follow_stream.rs

```diff
@@ -148,6 +148,14 @@ impl<Hash> Stream for FollowStream<Hash> {
                             continue;
                         }
                         Poll::Ready(Err(e)) => {
+                            // Re-start if a reconnecting backend was enabled.
+                            if e.is_disconnected_will_reconnect() {
+                                this.stream = InnerStreamState::New;
+                                return Poll::Ready(Some(Ok(FollowStreamMsg::Event(
+                                    FollowEvent::Stop,
+                                ))));
```

Comment:
Nit; you could do this instead here and below:

```rust
if e.disconnected_will_reconnect() {
    // Restart if a reconnecting RPC client is used, and gets disconnected.
    this.stream = InnerStreamState::Stopped;
    continue;
}
```

We do that in a couple of other places already so it'd be a bit more consistent :)